packages/salesforcedx-vscode-apex/package.json

```diff
@@ -141,6 +141,11 @@
           "type": ["string", "null"],
           "default": null,
           "description": "%java_home_description%"
+        },
```

Comment:
Let's go with this for now since we only have rename. Once we have others, we can decide what level of granularity. The only downside is that the name, i.e., `salesforcedx-vscode-apex.enable-refactor` will be kept in their settings.